js/wshop.js

```diff
@@ -333,22 +410,54 @@ var wshop = {
 
         });
 
+        // find anything with data-template
+        $productBlock.find('*[data-template]').each(function(){
+
+            // find data-template value
+            var templateName = jQuery(this).attr('data-template');
+
+            // render template
+            wshop.renderTemplate.bind(this)(templateName, product);
+
+            // callbacks
+            jQuery(document).trigger('wshop.templateRendered', [ jQuery(this) ]);
+        });
+
+        // Trigger "all rendered" event
+        $productBlock.trigger('wshop.allBlocksRendered');
+
     },
 
     handleAddToCart: function(){
```

Comment:
This should probably be named something more verbose, maybe "addSingleToCart" or something. Seems confusing to have "handleAddToCart" and "addToCart"